README.md

```diff
@@ -1,2 +1,43 @@
 # django-see-profile
 A simple middleware to capture useful profiling information in Django.
+
+## Requirements
+
+* Django 1.10+
+* django-security 0.10.0+
```

Comment:
Looks like the requirements here are different from `install_requires` in `setup.py`

Reply:
Whoops, didn't see Phi beat me to the punch on this on